src/descriptors/power_system_structs.json

```diff
@@ -1497,10 +1497,12 @@
         "validation_action": "error"
       },
       {
-        "name": "_forecasts",
-        "null_value": "InfrastructureSystems.Forecasts()",
-        "data_type": "InfrastructureSystems.Forecasts",
-        "default": "InfrastructureSystems.Forecasts()"
```

Comment:
The test is failing because you removed forecasts storage container from StaticReserve.  The raw files for the RTS system have time series data for this type.  Was that intentional?

Also, we could detect this error and display a helpful message.  I'll do that separately.

Reply:
The change was intentional. The idea is to make the definitions consistent #334, what we had defined as "Static" is in reality a Dynamic reserve and proportional never made too much sense. So replaced Static with dynamic and created a new Static reserve that has no forecast. 

The display message isn't clear. It just throws a MethodError